docs/android/standalone/implementationSteps.md

```diff
@@ -177,3 +179,84 @@ CartrawlerSDK.start(
     )
 )
 ```
+
+---
+
+### Start with Deep Link
+{: .no_toc }
+
+It is possible to launch the Car Rental SDK by using a url, it is up to you to decide where this url 
+will come from: a push notification or a native deep link.
+
+The url is pretty standard and should have the following pattern:
+
+```java
+schema://host?param1=123&param2=abcd&paramN=xyz
+
+// Example deep linking to the Search Results screen        
+yourschema://ct-car-rental?type=search-result&client_id=your-client-id&pt=2023-08-18T10:00:00Z&dt=2023-08-20T10:00:00Z&pkIATA=DUB&doIATA=ORK
```

Comment:
Think it might be worth adding somewhere that 'yourschema' is arbitratry and can be made up by the partner.